src/legos/typography/Heading.js

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import styled from "styled-components";
+import { Heading as LibHeading } from "grommet";
+
+const getFontSize =(level)=> {
+  switch (level) {
+    case 1:
+      return '54px'
+      break;
```

Comment:
I guess you don't need `break` after return 😬. This is what my VS Code says me:
![image](https://user-images.githubusercontent.com/12502321/82296370-c5844180-99b9-11ea-8995-8171860cdb9c.png)

Please remove all of `break` statements that follow `return` statement in this file 🙂.